### PR TITLE
Feature/create_mongo_db-GDEV-115

### PR DESCRIPTION
**Pull request by alamgirmani**

### CODE REVIEW

File: .idea/inspectionProfiles/Project_Default.xml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<component name="InspectionProjectProfileManager">
```
Comment (deepakunni3) [inline .idea/inspectionProfiles/Project_Default.xml:1]: Can we remove IntelliJ specific configurations? Looks like `.idea` folder was included in the repo.

File: README.md [hunk 2]
```diff
@@ -1,17 +1,37 @@
-# sandbox-request
+# Sandbox Sandbox Service
```
Comment (deepakunni3) [inline README.md:1]: Typo?

File: sandbox_request/__init__.py [hunk 3]
```diff
@@ -0,0 +1,19 @@
+"""
```
Comment (deepakunni3) [inline sandbox_request/__init__.py:1]: Perhaps it would be neater to have the module docstring after the license text.

**Review by deepakunni3 — CHANGES_REQUESTED** [review]
Looks good @alamgirmani ! Just a few comments above.

File: sandbox_request/__init__.py (hunk 3 above)
Comment (alamgirmani) [reply sandbox_request/__init__.py]: moved the docstrings below the licence text

File: README.md (hunk 2 above)
Comment (alamgirmani) [reply README.md]: fixed the typo

File: .idea/inspectionProfiles/Project_Default.xml (hunk 1 above)
Comment (alamgirmani) [reply .idea/inspectionProfiles/Project_Default.xml:1]: Removed the Intellij files and folders

File: sandbox_request/__init__.py (hunk 3 above)
Comment (deepakunni3) [reply sandbox_request/__init__.py]: I was suggesting the change for all python files, not just `__init__` 😅

**Review by deepakunni3 — APPROVED** [review]

**Review by KerstenBreuer — APPROVED** [review]
Great. If we can integrate the suggestions from #4, then this can be merged

File: .devcontainer/docker-compose.yml [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+      - 27017:27017
+
+  rabbitmq:
+      image: rabbitmq:3-management
+      container_name: 'rabbitmq'
+      hostname: 'rabbitmq'
+
+      # (Adding the "ports" property to this file will not forward from a Codespace.)
+      ports:
+        - 5672:5672
```
Comment (ac-jorellanaf) [inline .devcontainer/docker-compose.yml:51]: Dont think these lines do anything.

**Pull request merged**